client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -0,0 +1,48 @@
+{{#let @form as |Form|}}
+  <Ui::Question class="fieldset relative scale-fade-in" as |Q|>
+    <Q.legend>
+      <strong>
+        <span class="label">
+          {{~@landUseAction.dcpZoningresolutiontype~}}
+          {{#if (lookup-action-type @landUseAction.dcpZoningresolutiontype)}}
+            <LabsUi::IconTooltip
+              @tip={{lookup-action-type @landUseAction.dcpZoningresolutiontype}}
+              @icon="info-circle"
+              @side="bottom"
+              @transform="shrink-4 up-2"
+              @fixedWidth={{true}}
+            />
+          {{/if}}
+        </span>
+        {{#if (in-array this.actionsWithSectionNumber @landUseAction.dcpZoningresolutiontype)}}
+          {{@landUseAction.dcpZrsectionnumber}}
+        {{/if}}
+        {{#if (and (eq @landUseAction.dcpZoningresolutiontype 'ZR') (eq @landUseAction.dcpZrsectionnumber 'Appendix F'))}}
+          {{@landUseAction.dcpZrsectionnumber}}
+        {{/if}}
+      </strong>
+    </Q.legend>
+    {{#if (in-array this.actionsWithModifiedZrSectionNumberQuestion @landUseAction.dcpZoningresolutiontype)}}
+      <label class="action-form-label">
+        Which sections of the Zoning Resolution does this modify?
+        <small class="help-text display-block">
+          Provide the Zoning Resolution section number(s). Ex. ZR Sec. 42-10 and 43-17
+        </small>
+          <Form.Field
+            @attribute="dcpModifiedzrsectionnumber"
+            as |TextInput|
+          >
+            <TextInput
+              type="text"
+              maxlength="250"
+              data-test-input="dcpModifiedzrsectionnumber"
+            />
+          </Form.Field>
+      </label>
```

Comment:
Can this use the question component? Does the `. action-form-label` actually do anything?